src/helpers/validate-request-query.js

```diff
@@ -52,6 +53,18 @@ function validateRequestQuery(requestQuery) {
   }
 };
 
+function parseRequestQuery(requestQuery) {
+  try {
+    const parsedRequestQuery = {
+      ...requestQuery,
+      geometry: typeof requestQuery.geometry === 'string' ? JSON.parse(requestQuery.geometry) : requestQuery.geometry
```

Comment:
I think this is going to cause you some problems.  There are valid `geometry` string values that are not stringified JSON.  Example: a query parameter `geometry=10,10,10,10` will arrive in this code as `"10,10,10,10"`.  Your code will try to JSON.parse this and fail I think.

An approach that is often used in this situation is:
```js
function tryJsonParse (input) {
  try {
    return JSON.parse(input);
  } catch (err) {
    this.log.debug(err);
    return input;
  }
}
```

Reply:
Now, should validate both geometry object and geometry comma separated string value.